controllers/helpers.go

```diff
@@ -607,10 +607,7 @@ func ShouldDeleteIndividualResources(ctx context.Context, clusterScope *scope.Cl
 		return true
 	}
 	grpSvc := groups.New(clusterScope)
-	managed, err := grpSvc.IsManaged(ctx)
-	// Since this is a best effort attempt to speed up delete, we don't fail the delete if we can't get the RG status.
-	// Instead, take the long way and delete all resources one by one.
-	return err != nil || !managed
+	return grpSvc.ShouldDeleteIndividualResources(ctx)
```

Comment:
```suggestion
	return groups.New(clusterScope).ShouldDeleteIndividualResources(ctx)
```

Since `groups.New()` doesn't return an error